chapters.md

```diff
@@ -3,31 +3,84 @@
 
 # Chapters
 
-## Edition and Chapter Flags
+The Matroska Chapters system can have multiple `Editions` and each `Edition` can consist of `Simple Chapters` where a chapter start time is used as marker in the timeline only. An `Edition` can be more complex with `Ordered Chapters` where a chapter end time stamp is additionally used or much more complex with `Linked Chapters`. The Matroska Chapters system can also have a menu structure, borrowed from the DVD menu system, or have it's own Native Matroska menu structure.
```

Comment:
The term `Simple Chapters` is simple not well, it should be `Basic Chapters`.
https://github.com/hubblec4/Matroska-Playback/blob/master/src/BasicChapters.md

Is it a good idea to describe the chapter types in front of this document?
https://github.com/hubblec4/Matroska-Playback/blob/master/src/NestedChapters.md
https://github.com/hubblec4/Matroska-Playback/blob/master/src/OrderedChapters.md
https://github.com/hubblec4/Matroska-Playback/blob/master/src/NestedOrderedChapters.md

Reply:
I would prefer changing the term `Simple Chapters` in another PR. It's currently found in `EditionFlagOrdered`. You can open an issue for that.

Reply:
IMO Nested chapters are not a special kind of chapters. So they shouldn't be mentioned in the same paragraph which described different ways the chapters can be interpreted.